molecule/default/tests/test_default.py

```diff
@@ -5,10 +5,28 @@
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
 
+conf_md5 = '87457a8be8e47b51d03da3154f4a1373'
+conf_sha = '897215c11150201799b8c4d6e1605ab359904bfab85f39e2da2480752b55c3e8'
 
-def test_hosts_file(host):
-    f = host.file('/etc/hosts')
 
-    assert f.exists
-    assert f.user == 'root'
-    assert f.group == 'root'
+def test_haproxy_is_installed(host):
+    haproxy = host.package('haproxy')
+    assert haproxy.is_installed
+    assert haproxy.version.startswith('1.5')
```

Comment:
I am no quite sure that is a good idea to precise the version. Especiallly with an old one.

Reply:
It reflects the current state of the code as well as our test targets. I think is good release 1.3.0 and can be removed later when we will test against more recent versions of haproxy.

Reply:
Looks good to me.